### DISCUSSION TITLE
Stash needing to be re-enabled upon startup/update/reload

### DISCUSSION THREAD

**Discussion by alanjwn** [rxhanson/RectanglePro-Community#503] [Q&A]
Hi,

I'm not sure if it's just me or the way my computer is configured but it has happened many times on my 2 macOS devices.

**Each time I start up or restarted--or just right now, when the application just updated itself--I've had to go into the Preferences and uncheck and check the "Enable Stash" option again in order to use the keyboard hotkeys (which are customized) again to use the feature.**

Is this by design? A bug? Or some other conflicting setting on my Mac?

Just wanted to bring some attention to this issue. Not looking for an immediate fix.

Thanks! Amazing app :) ⭐️

![2024-05-09 at 13 25 25](https://github.com/rxhanson/RectanglePro-Community/assets/72910212/3081ca40-fbf1-419b-854b-573f13ac4ac4)

**Comment by rxhanson** [comment]
Thanks for reporting. I think I may have introduced this bug a few releases ago. I'm planning to push out a release this weekend and will see if I can fit it into that one.
